src/lt/objs/find.cljs

```diff
@@ -215,8 +215,8 @@
 
 (object/add-behavior! line-input ::exec-active!)
 
-(cmd/command {:command :goto-line
-              :desc "Editor: Goto line"
+(cmd/command {:command :go-to-line
+              :desc "Editor: go to line"
```

Comment:
This line should have a capital 'G' for consistency... so `:desc "Editor: Go to line"`